README.md

```diff
@@ -4,14 +4,35 @@ This plugin helps to reformat existing codebase to adopted CodeStyle and work in
 
 ## Configuration steps
 
-1. Add this to your main ` build.gradle ` file : This should fail build operation if there exists codestyles adherence error(s).
+1. Adapt your ` build.gradle ` file to the following structure : This should fail build operation if there exists codestyles adherence error(s).
 ```
-apply plugin: 'checkstyle'
+   allprojects {
+    apply plugin: 'checkstyle'
 
     checkstyle {
-        config project.resources.text.fromUri('https://raw.githubusercontent.com/oyewaleoyelami/check/master/checkstyle.xml')
-        toolVersion = "8.11"
-    }
+               config project.resources.text.fromUri('https://raw.githubusercontent.com/UvitaTeam/code-style/master/vivyjava/src/main/resources/META-INF/file/checkstyle.xml')
+               toolVersion = "8.12"
+           }
+
+    repositories {
+           jcenter()
+           maven { url 'https://jitpack.io'}
+       }
+
+    dependencies {
+               dependency 'com.puppycrawl.tools:checkstyle:8.12'
+               dependency 'com.github.UvitaTeam.code-style:checkstyle-custom:enhance-SNAPSHOT'
+         }
+
+     }
+
+   subprojects {
+
+        dependencies {
+            compile 'com.puppycrawl.tools:checkstyle'
```

Comment:
once more - it should not be for every subproject, only in the checkstyle-custom

Reply:
removed but are suggesting I removed place the left dependency under that of allprojects ?